.github/workflows/release.yaml

```
@@ -11,48 +8,78 @@ on:
+
+      - name: Get the release version from the package
+        shell: bash
+        run: echo "VERSION=$(cargo tree | grep keepass-2-file | cut -d' ' -f2)" >> $GITHUB_ENV
 
-      - name: Create Release
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build
+        uses: actions-rs/cargo@v1
```

actionlint (1.7.4)

the runner of "actions-rs/cargo@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)